http_client.rst

```diff
@@ -731,6 +731,59 @@ The ``RetryableHttpClient`` uses a
 decide if the request should be retried, and to define the waiting time between
 each retry.
 
+Retry over different uris
+.........................
+
+.. versionadded:: 6.3
+    The multiple ``base_uri`` feature was added in Symfony 6.3.
+
+The ``RetryableHttpClient`` can be configured to use multiple base URIs. This
+feature provides increased flexibility and reliability for making HTTP
+requests. Pass an array of base URIs as option ``base_uri`` when making a 
+request::
+
+    $response = $client->request('GET', 'foo-bar', [
+        'base_uri' => [
+            // first request will use this base URI
+            'http://example.com/a/',
+            // if first request fails, this second base URI will be used
+            'http://example.com/b/',
+        ],
+    ]);
+
+When the number of retries is higher than the number of base URIs, the 
+last base URI will be used for remaining retries.
+
+If you want to shuffle the order of base URIs for each retry attempt, nest the
+base URIs you want to shuffle in an additional array::
+
+    $response = $client->request('GET', 'foo-bar', [
+        'base_uri' => [
+            'http://example.com/a/',
+            [
+                // one random URI from this list will be used on retry #2
+                'http://example.com/b/',
+                'http://example.com/c/',
+            ],
+            // non-nested base URIs are used in order
+            'http://example.com/d/',
+        ],
+    ]);
+
+This feature allows for a more randomized approach to handling retries, 
```

Comment:
```suggestion
This feature allows for a more randomized approach of handling retries, 
```

Reply:
I think "to" is more correct in English here

Reply:
Yeah not sure about this one either